chapter_computational-performance/hardware_vn.md

```diff
@@ -570,7 +582,12 @@ Matching both goals is an area of active research.
 See e.g., [DGL](http://dgl.ai), a library tuned for deep learning on graphs.
 -->
 
-*dịch đoạn phía trên*
+Đương nhiên khi tối ưu cho quá trình tính toán, ta bắt buộc phải có một số thoả hiệp nhất định.
+Một trong số đó là GPU không xử lý tốt dữ liệu ngắt quãng hoặc thưa thớt.
+Trừ một số ngoại lệ đáng chú ý, ví dụ như [Gunrock](https://github.com/gunrock/gunrock) :cite:`Wang.Davidson.Pan.ea.2016`,
+mô hình truy cập ma trận thưa và vector không phù hợp với các thao tác đọc đột ngột khi băng thông cao, trường hợp mà GPU trội hơn.
```

Comment:
```suggestion
mô hình truy cập ma trận thưa và vector không phù hợp với các thao tác đọc theo cụm (*burst read*) khi băng thông cao mà GPU trội hơn.
```
Theo https://stackoverflow.com/questions/6486062/burst-mode-definition
>Burst mode is when you send one address to the memory, but rather than reading/write the data only for the specified address, you also read/write some number of consecutive locations(typically 4 or 8).